### PR TITLE
Add secure development guidance for Express quickstart

### PR DESCRIPTION
**Pull request by joshcanhelp**

### CODE REVIEW

**Review by stevehobbsdev — DISMISSED** [review]

File: articles/quickstart/webapp/express/01-login.md [hunk 1]
```diff
@@ -14,30 +14,72 @@ useCase: quickstart
...
+```js
+const fs = require('fs');
+const key = fs.readFileSync('./localhost-key.pem');
+const cert = fs.readFileSync('./localhost.pem');
+
+const express = require('express');
+const https = require('https');
+https.createServer({ key: key, cert: cert }, express()).listen('3000', () => {
+  console.log('Listening on https://localhost:3000');
+});
```
Comment (davidpatrick) [inline articles/quickstart/webapp/express/01-login.md:72]:
```suggestion
const express = require('express');
const https = require('https');
const fs = require('fs');

const key = fs.readFileSync('./localhost-key.pem');
const cert = fs.readFileSync('./localhost.pem');

https.createServer({ key, cert }, express()).listen('3000', () => {
  console.log('Listening on https://localhost:3000');
});
```

Comment (davidpatrick) [reply articles/quickstart/webapp/express/01-login.md:72]: Not sure how to do a suggested change for a block... :hmm:

File: articles/quickstart/webapp/express/01-login.md [hunk 2]
```diff
@@ -14,30 +14,73 @@ useCase: quickstart
...
+Finally, copy the following fields for your application for use in step 7:
+
+* **Domain**
+* **Client ID**
+
+## Secure Local Server
+
+Applications that handle sensitive data must be served over secure channels. This includes local applications as they may handle the same sensitive data and should be built as close to production-ready as possible.
+
+Complete instructions are below or see our [Secure Local Development guide](https://auth0.com/docs/libraries/secure-local-development) for additional explanation.
```
Comment (davidpatrick) [inline articles/quickstart/webapp/express/01-login.md]:
```suggestion
Complete instructions are below or see our [Secure Local Development guide](/libraries/secure-local-development) for additional explanation.
```

File: articles/quickstart/webapp/express/01-login.md [hunk 3]
```diff
@@ -14,30 +14,73 @@ useCase: quickstart
...
+If you do not already have a method of generating local certificates, [install `mkcert`](https://github.com/FiloSottile/mkcert#installation) and run the following commands in your terminal to generate a certificate and key for this application:
+
+```bash
+#!/bin/sh
+❯ cd /path/to/application/root
+❯ mkcert -install
+❯ mkcert localhost
+```
+
+### 5. Install Certificates
```
Comment (davidpatrick) [inline articles/quickstart/webapp/express/01-login.md]: Here https://github.com/auth0/docs/pull/8668/files#diff-c4d25e0f70b3c3c4e685cd9645275de7R60

We say `Serve the SSL certificate`, what do you think about having consistent verbiage here?

Comment (davidpatrick) [inline articles/quickstart/webapp/express/01-login.md]:
```suggestion
### 5. Serve the SSL certificate
```

File: articles/quickstart/webapp/express/01-login.md [hunk 4]
```diff
@@ -14,30 +14,73 @@ useCase: quickstart
...
+* **Domain**
+* **Client ID**
+
+## Secure Local Server
+
+Applications that handle sensitive data must be served over secure channels. This includes local applications as they may handle the same sensitive data and should be built as close to production-ready as possible.
+
+Complete instructions are below or see our [Secure Local Development guide](/libraries/secure-local-development) for additional explanation.
+
+### 4. Generate Certificates
```
Comment (davidpatrick) [inline articles/quickstart/webapp/express/01-login.md]:
```suggestion
### 4. Generate an SSL Certificate
```

**Review by davidpatrick — APPROVED** [review]

**Pull request merged**